docs/permissions.rst

```diff
@@ -744,60 +744,71 @@ are as follows:
-|effective_permissions|_ always imply |inherited_permissions|_ (i.e.: using both query simultaneously is redundant).
-The reason why we obtain these sets for cases :sup:`(2)` is also because there is no other :term:`Permission` applied
-to any of their parent :term:`Service` or :term:`Resource`. Contrarily, ``resource-A`` :sup:`(3)` now additionally
-receives :term:`Permission` ``read`` indirectly from its parent ``service-2`` (note: ``write`` is redundant here).
-Similarly, ``resource-B2`` :sup:`(4)` which did not even have any immediate :term:`Permission` applied to it,
-now receives both ``read`` and ``write`` access, respectively from its parents ``resource-B1`` and ``service-3``. This
-demonstrates why, although |effective_permissions|_ imply |inherited_permissions|_, they do not necessarily resolve to
-the same result according to the effective :term:`Resource` hierarchy and its parent-children resolution implementation.
+In this case, all :term:`Resource` entries that had :term:`Permission` directly set on them, whether through
+:term:`User` :sup:`(1)` or :term:`Group` :sup:`(2)` combination, all return the exact same set of :term:`Permission`.
+This is because |effective_permissions|_ always imply |inherited_permissions|_
+(i.e.: using both query simultaneously is redundant).
+The reason why we obtain these sets for cases :sup:`(1)` and :sup:`(2)` is because there is no
+other :term:`Permission` applied  to any of their parent :term:`Service` or :term:`Resource`.
+Contrarily, ``resource-A`` :sup:`(3)` now additionally (in contrast to ``inherited=true``)
+receives :term:`Permission` ``write`` indirectly from its parent ``service-2``.
+This is due to the ``recursive`` hierarchical inheritance of the parent-children *permission scheme*.
+Furthermore, ``resource-A`` :sup:`(3)` preserves the ``read`` :term:`Permission` that was directly
+applied for ``example-user``. Similarly, ``resource-B2`` :sup:`(4)` which did not even have any
+immediate |applied_permissions|_ on it, now receives both ``read`` and ``write`` access, respectively
+from its parents ``resource-B1`` (through inherited :term:`Group` :term:`Permission <Permissions>`)
+and ``service-3`` (through direct :term`User` :term:`Permission <Permissions>`).
+This demonstrates why, although |effective_permissions|_ imply |inherited_permissions|_,
+they do not necessarily resolve to the same result according to the effective :term:`Resource` hierarchy
+and its parent-children resolution implementation.
 
 Using ``effective`` query tells `Magpie` to rewind the :term:`Resource` tree from the requested :term:`Resource` up to
 the top-most :term:`Service` in order to accumulate all |inherited_permissions|_ observed along the way for every
-encountered element. All :term:`Permission` that is applied *higher* to the requested :term:`Resource` are considered
+encountered element. All :term:`Permission` that is applied *higher* that the requested :term:`Resource` are considered
```

Comment:
I would maybe correct this :
All :term:\`Permission\<Permissions\>\` that are applied \*higher\* **than** the ...